docs/source/install/deb.rst

```diff
@@ -30,13 +30,12 @@ Install MongoDB, RabbitMQ, and PostgreSQL.
 Setup repositories
 ~~~~~~~~~~~~~~~~~~~
 
-  .. code-block:: bash
+The following script will detect your platform and architecture and setup the repo accordingly. It'll also install the GPG key for repo signing.
```

Comment:
Also, please add a line not about deprecation of bintray and point to migration script in https://github.com/StackStorm/st2-packages/pull/176?

We should mirror all these changes to v1.3 branch as well.